changelogs/fragments/junos_routing_options_resource_module.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
With this PR, we should be able to deprecate the options from the other modules and add a `deprecated_features` section here.

Reply:
+1